engine/player/unique_gear_shadowlands.cpp

```diff
@@ -3422,6 +3422,53 @@ void elegy_of_the_eternals( special_effect_t& effect )
   std::rotate( vec->rbegin(), vec->rbegin() + 1, vec->rend() );
 }
 
+// id=367336 - Brood of the Endless Feast	(driver - damage on effect 1)
+// id=368585 - Scent of Souls (target debuff)
+// id=368587 - Rabid Devourer Chomp (physical damage at max stacks)
+void bells_of_the_endless_feast( special_effect_t& effect )
+{
+  struct rabid_devourer_chomp_t : generic_proc_t
+  {
+    rabid_devourer_chomp_t( const special_effect_t& effect )
+      : generic_proc_t( effect, "rabid_devourer_chomp", effect.player->find_spell( 368587 ) )
+    {
+      base_dd_min = base_dd_max = effect.driver()->effectN( 1 ).average( effect.item );
+    }
+  };
+
+  struct brood_of_the_endless_feast_cb_t : dbc_proc_callback_t
+  {
+    action_t* damage;
+
+    brood_of_the_endless_feast_cb_t( const special_effect_t& effect )
+      : dbc_proc_callback_t( effect.player, effect ),
+        damage( create_proc_action<rabid_devourer_chomp_t>( "rabid_devourer_chomp", effect ) )
+    {
+    }
+
+    void execute( action_t* a, action_state_t* s ) override
+    {
+      dbc_proc_callback_t::execute( a, s );
+
+      actor_target_data_t* td = a->player->get_target_data( s->target );
+
+      if ( td->debuff.scent_of_souls->at_max_stacks() )
+      {
+        td->debuff.scent_of_souls->expire();
+        damage->set_target( s->target );
+        damage->execute();
+      }
+      else
+      {
+        double stacks = rng().range( 5, 7 );
+        td->debuff.scent_of_souls->execute( stacks );
```

Comment:
```suggestion
        td->debuff.scent_of_souls->trigger( stacks );
```
Or a comment why we are going straight to `execute()`.